### DISCUSSION TITLE
Updating from v2.3.3 to 2.4.7 causes a critical error on website

### DISCUSSION THREAD

**Discussion by carlbannister** [liedekef/events-made-easy#277] [Q&A] [answered]
Hi I have tried to update from version 2.3.3 to 2.4.7 on my staging site but it causes a critical error. I know this is a big jump in versions but should it make a difference? I have let the versions lapse as our venue has been shut for over a year so no need for a booking system or website for that matter.
It is now being resurrected so need to get it up to date and secure.
I am not sure how to start debugging this as just an armature, can you give me some pointers of where to start.

Many Thanks

Carl

**Comment by liedekef** [comment]
Just make sure to use at least php version 8. Version 7.4 and older are end of life since the beginning of this year.

**Reply by carlbannister** [reply, marked as answer]
Ive just come to that conclusion as well, will give it a try, thanks for the swift response
